mscore/pagesettings.cpp

```diff
@@ -81,9 +88,46 @@ PageSettings::PageSettings(QWidget* parent)
 
 PageSettings::~PageSettings()
       {
+      QSettings settings;
+
```

Comment:
I think he is referring to having these "remember window resize" functionality in a similar manner in all the windows. I did it in a similar manner to start center(https://github.com/musescore/MuseScore/blob/master/mscore/startcenter.cpp) but my code still have some improvements to be done.(See the comments in #2500)